bin/index.cjs

```diff
@@ -0,0 +1,50 @@
+#! /usr/bin/env node
+const chalk = require('chalk');
+const { execFileSync } = require('child_process');
+const shell = require('shelljs');
+
+const SUCCESS_CODE = 0;
+const ERROR_CODE = 1;
+
+function exitOnError(errorMessage) {
+  shell.echo(prettifyErrorMessage(errorMessage));
+  shell.exit(ERROR_CODE);
+}
+
+function prettifyStepTitle(title) {
+  return chalk.bgBlue.bold(title);
+}
+
+function prettifyErrorMessage(errorMessage) {
+  return chalk.bgRed.bold(errorMessage);
+}
+
+
+if (!shell.which('git')) {
+  exitOnError('Sorry, this script requires git');
+}
+
+if (!shell.which('degit')) {
```

Comment:
To avoid this req we can do something like `git clone --depth 1` and delete `.git` folder